### ISSUE TITLE
Performance issues with ddrnet

### ISSUE THREAD

**Issue by mhmd-mst** [sithu31296/semantic-segmentation#32]
Hello, I am using ddrnet , i tried several other models with the same dataset i am working on and they improve over time except ddrnet after 10 epoch the miou starts decreasing , Does that mean ddrnet perform bad or is something wrong with the code i should fix?

**Comment by sithu31296** [comment]
That is strange. I trained all light-weight models in my repo on face parsing datasets and among them, DDRNet has the best accuracy. Wait for longer epochs and if it is still lower than other models, you can change `align_corners` to `True` in 'ddrnet.py'. And set `aux_weights` to `[1, 0.5]`. Furthermore, try to tune hyperparameters.

**Comment by mhmd-mst** [comment]
I tried 50 epochs so maybe it is overfitting right?

**Comment by sithu31296** [comment]
Observe the other models' behavior and see if it is similar. It also depends on your dataset size.

**Comment by mhmd-mst** [comment]
I tried bisenetv1 with resnet18 and sfnet with resnetd18 both keep increasing, my dataset is 16k

**Comment by sithu31296** [comment]
Your dataset size is already large. Try to train more epochs and do the above tricks I mentioned.
